package.json

```diff
@@ -104,12 +104,12 @@
     "nav-frontend-skjema-style": "^2.2.3",
     "nav-frontend-spinner": "^2.0.17",
     "nav-frontend-spinner-style": "^1.0.2",
-    "nav-frontend-tabell-style": "^2.0.2",
+    "nav-frontend-tabell-style": "^2.1.1",
     "nav-frontend-tabs": "^2.0.1",
     "nav-frontend-tabs-style": "^2.0.1",
     "nav-frontend-typografi": "^3.1.1",
     "nav-frontend-typografi-style": "^1.1.1",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^3.0.0",
```

Comment:
Trengte vi ikke å gjøre noen endringer for å oppdatere denne?
https://github.com/node-fetch/node-fetch/blob/HEAD/docs/v3-UPGRADE-GUIDE.md